packages/blockchain-wallet-v4-frontend/src/modals/ActiveRewards/WithdrawalForm/CustodialAccount/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Flex, Padding, SemanticColors, Text } from '@blockchain-com/constellation'
+
+import { Icon } from 'blockchain-info-components'
+
+import { Wrapper } from './CustodialAccount.model'
+import { PropsType } from './CustodialAccount.types'
+
+const CustodialAccount = ({ coin, cryptoAmount, fiatAmount, product }: PropsType) => {
+  const { coinfig } = window.coins[coin]
```

Comment:
I suggest we try to avoid using `window` directly. There is `getCoins` selector in `packages/blockchain-wallet-v4/src/redux/data/coins/selectors.ts`.